### PR TITLE
[179, 182, 180, 181, 178, 185, 184] bug fixes

### PR DESCRIPTION
**Pull request by khasanrashidov**
### Where

[NL-179 Sign Up flow - unclear error message](https://jjbcapital.atlassian.net/browse/NL-179)
[NL-182 Add lot profile link in first message sent from seller (automatic message)](https://jjbcapital.atlassian.net/browse/NL-182)
[NL-180 View lot profile (buyer) - disable 'Place a bid" button in case the lot is not active](https://jjbcapital.atlassian.net/browse/NL-180)
[NL-181 Conversations are not sorted after message sent/received](https://jjbcapital.atlassian.net/browse/NL-181)
[NL-178 Add a time display for sent messages](https://jjbcapital.atlassian.net/browse/NL-178)
[NL-185 Inspect the search behavior (if it really searches all available lots for buyer)](https://jjbcapital.atlassian.net/browse/NL-185)
[NL-184 Going to seller profile through lot profile](https://jjbcapital.atlassian.net/browse/NL-184)

### What

The goal of the changes is to fix all of the specified issues.

### How

Every issue has been resolved.

### Affected Area

Auctionify.API.ClientApp
Auctionify.Application

### PR Checklist

_Build_

- [x] Build Succeeded

_Unit Tests_

- [x] All Tests Passed, 0 Ignored, 0 Failed

### CODE REVIEW

**Review by SaidganiUsm — APPROVED** [review]

**Comment by khasanrashidov** [comment]
resolved, finished

**Review by SaidganiUsm — APPROVED** [review]

File: src/Auctionify.API/ClientApp/src/app/components/general/chat/chat-item/chat-item.component.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h4 class="chat__top-nickname">
                     }}
                 </h4>
                 <p class="chat__top-time">
-                    {{ messageDate | date : 'hh:mm aa' }}
+                    {{ messageDate | date : 'HH:mm' }}
```
Comment (TsvirkoNikita) [inline src/Auctionify.API/ClientApp/src/app/components/general/chat/chat-item/chat-item.component.html:25]: Make sense to add also date here

**Review by TsvirkoNikita — CHANGES_REQUESTED** [review]

File: src/Auctionify.Application/Scheduler/Jobs/FinishLotJob.cs [hunk 2]
```diff
@@ -98,7 +108,10 @@ public async Task Execute(IJobExecutionContext context)
 						SenderId = lot.SellerId,
 						ConversationId = conversation.Id,
 						Body =
-							$"Congratulations, Dear Buyer! You won the auction for lot with id: {lotId}!",
+							$"Congratulations, Dear Buyer! "
+							+ $"My name is {lot.Seller.FirstName} {lot.Seller.LastName} and I am the seller of the lot with id: {lotId}. "
```
[the remark this answers is not in the document]
Comment (khasanrashidov) [reply src/Auctionify.Application/Scheduler/Jobs/FinishLotJob.cs:112]: Right!) will fix all issues

File: src/Auctionify.API/ClientApp/src/app/components/general/lot-profile/lot-profile.component.html [hunk 3]
```diff
@@ -210,7 +210,8 @@ <h2>
                             [disabled]="
                                 lotData.bids.length == 0 ||
                                 lotData.bids.at(0)?.buyerId !==
-                                    this.currentUserId
+                                    this.currentUserId ||
+                                lotData.lotStatus.name !== 'Active'
```
[the remark this answers is not in the document]
Comment (khasanrashidov) [reply src/Auctionify.API/ClientApp/src/app/components/general/lot-profile/lot-profile.component.html]: fixed

**Review by TsvirkoNikita — APPROVED** [review]

**Pull request merged**